src/main/java/transforms/GeographyPointTransform.java

```diff
@@ -51,7 +51,20 @@ public Object apply( Map<String, String> row ) {
         lon = Parsers.parseDouble( lon );
 
         if (  lat != null && lon != null  ) {
-            return lat.toString() + "," + lon.toString();
+            String latitude = lat.toString();
```

Comment:
This looks like the same logic as above -- it would be cleaner / easier to maintain this if you write a static `String getFormattedLatLon( Object latitude, Object longitude) { ... }` and then call that from both places